packages/console/src/pages/Profile/utils.ts

```diff
@@ -39,3 +43,26 @@ export const popupWindow = (url: string, windowName: string, width: number, heig
     ].join(',')
   );
 };
+
+export const handleError = async (
+  error: unknown,
+  exec?: (errorCode: string, message: string) => Promise<void>
+) => {
+  if (error instanceof HTTPError) {
+    const logtoError = await error.response.json<RequestErrorBody>();
+    const { code, message } = logtoError;
+
+    if (exec) {
+      await exec(code, message);
+
+      return;
```

Comment:
this will swallow unhandled errors?

Reply:
Yes, because the error has already been handled inside the `exec` function. Therefore, no more error handling is needed.
But for the case that non `exec` is provided, we will use the rest of the handling logic to take care of it, either show error in toast or throw it to the ErrorBoundary

Reply:
You're right. There was a flaw in the logic. updated now